BattleSim/src/main/java/com/revature/controllers/AccountController.java

```diff
@@ -0,0 +1,122 @@
+package com.revature.controllers;
+
+import java.io.IOException;
+import java.util.List;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+
+import com.google.gson.Gson;
+import com.revature.models.Users;
```

Comment:
In the most recent push to UserService I believe the unnecessary plurals were removed, so addUsers is now addUser, deleteUsers is deleteUser etc.